prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1397,6 +1360,47 @@ struct fi_ops_cm rxr_ep_cm = {
 	.join = fi_no_join,
 };
 
+/*
+ * @brief explicitly allocate a chuck of memory for 3 RX packet pools:
+ *     unexpeced (unexp), out-of-order (ooo), and read-copy
+ *
+ * @param ep[in]	endpoint
+ * @return		On success, return 0
+ * 			On failure, return a negative error code.
+ */
+int rxr_ep_grow_rx_pkt_pools(struct rxr_ep *ep)
```

Comment:
We have rx_pkt_efa_pool and rx_pkt_shm_pool created and allocated in ep initialization, will this naming introduce confusion to different behaviors of rx pkt pools?

Reply:
In new revision, I added call of `ofi_bufpool_grow` for these 2 packet pools in the function.

Reply:
These 2 packet pools are already implicitly "growed" in ep initialization, so the `ofi_bufpool_grow` here actually does not take effect?

Reply:
They are not growed in ep initialization. Noticing that I removed the call to `rxr_ep_bulk_post_rx_bufs()` in `rxr_ep_ctrl` (which does the EP initialization).

This call to grow indeed grow the two packet pool. If they are not grown here, they will be grown later in `rxr_ep_progress_post_rx_pks()` when `rxr_ep_bulk_post_rx_pkts()` is called.

However, it is better to grow them explicitly, this way we will be able to distinguish that case pkt_entry allocation failed, or packet pool failed.  (Without explicit call to grow, memory of packet pool is allocated when 1st call to `rxr_pkt_entry_alloc`.)

Reply:
I see, thanks!